DESCRIPTION

```diff
@@ -3,7 +3,12 @@ Type: Package
 Title: Test Data for the Pharmaverse Family of Packages
 Version: 0.0.2
 Authors@R: c(
-  person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = c("aut", "cre"))
+  person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = c("aut", "cre")),
+  person("Kangjie", "Zhang", role = "aut"),
+  person("Stefan", "Bundfuss", role = "aut"),
```

Comment:
please can you sort alphabetically below the maintainer, similar to what admiral and admiraldev are doing - thanks